.circleci/config.yml

```diff
@@ -563,12 +557,30 @@ jobs:
           study_key: fon
 
   app-build-and-store-job:
+    description: Run build and store archive of it
     working_directory: *ng_workspace_path
     executor:
       name: build-executor
+    parameters:
+      study_key:
+        type: string
+        default: << pipeline.parameters.study_key >>
+      store_build_artifacts:
+        type: boolean
+        default: true
```

Comment:
Adding new job parameter `store_build_artifacts`. I'm reusing this job for running tests on PR commits, and don't want to save any generated build artifacts that could be used in deploy to GAE.